pydis_site/static/css/home/index.css

```diff
@@ -107,6 +111,10 @@ h1 {
     margin: 0 1em;
 }
 
+[data-theme="dark"] #showcase .box {
+    box-shadow: 0 0.5em 1em -0.125em rgb(10 10 10 / 10%), 0 0px 0 1px rgb(10 10 10 / 2%);
```

Comment:
This was copied as-is from bulma's own code, only changing the shadow colors to match those of the project cards.